decode.go

```diff
@@ -157,6 +184,8 @@ func (dec *Decoder) Start(n int) error {
 	// start reading OSMData
 	go func() {
 		var inputIndex int
+		var blobHeader *OSMPBF.BlobHeader
```

Comment:
Is there any advantage to putting `blobHeader` and `blob` outside the scope of the subsequent for loop? Would they be better declared implicitly at `blobHeader, blob, err := dec.readFileBlock()`?